fe/fe-core/src/main/java/com/starrocks/qe/ConnectContext.java

```diff
@@ -133,9 +134,13 @@
 
     protected DumpInfo dumpInfo;
 
+    protected HiveMetaCache.ProfileStats hiveMetaCacheProfileStats;
+
     // The related db ids for current sql
     protected Set<Long> currentSqlDbIds = Sets.newHashSet();
 
+    protected int plannerRunningTime = 0;
```

Comment:
maybe `long` is better?

Reply:
better to take the unit in the name. Such as planRunningMs planRunningSeconds